src/component/menu/Footer/ApiDetails.tsx

```diff
@@ -0,0 +1,51 @@
+import { ReactElement } from 'react';
+import ConditionallyRender from '../../common/ConditionallyRender';
+import { IVersionInfo } from '../../../interfaces/uiConfig';
+import { formatCurrentVersion, formatUpdateNotification } from './ApiDetails.helpers';
+
+export interface IPartialUiConfig {
+	name: string;
+	version: string;
+	slogan?: string;
+	environment?: string;
+	versionInfo?: IVersionInfo;
+}
+
+interface IApiDetailsProps {
+	uiConfig: IPartialUiConfig;
+}
+
+export const ApiDetails = (props: IApiDetailsProps): ReactElement => {
+	const instanceId = props.uiConfig.versionInfo?.instanceId;
+	const currentVersion = formatCurrentVersion(props.uiConfig);
+	const environment = props.uiConfig.environment;
+	const updateNotification = formatUpdateNotification(props.uiConfig);
+	
+	return (
+		<section title="API details">
+			<h4>
+				{currentVersion}{' '}
+				<ConditionallyRender
+					condition={environment}
```

Comment:
you could automatically convert this to a boolean with the `!!`. I guess we need to decide on which pattern we prefer @FredrikOseberg 

```suggestion
					condition={!!environment}
```

Reply:
I usually prefer `Boolean(environment)` just because it's more explicit.

Reply:
True! I like `Boolean(foo)`. Matches `String(foo)`, `Number(foo)` etc.